functions:
    - $(find rosplan_demo_stage)/scripts/sampling_sensing_functions.py
topics:
    robot_at:
        params:
            - kenny
            - '*'
        topic: /amcl_pose
        msg_type: geometry_msgs/PoseWithCovarianceStamped

services:
    pedestal_visible_from:
        params:
            - '*'
            - '*'
        service: /rosplan_knowledge_base/state/instances
        srv_type: rosplan_knowledge_msgs/GetInstanceService
        time_between_calls: 2
        request: GetInstanceServiceRequest(type_name="waypoint")
